src/handlers/uniswap.ts

```diff
@@ -428,29 +418,43 @@ export const getAllTokens = async (excluded = []): Promise<AllTokenInfo> => {
   } catch (err) {
     logger.log('error: ', err);
   }
+
   data.forEach(token => {
     const tokenAddress = toLower(token.id);
     const metadata = getTokenMetadata(tokenAddress);
+
     const tokenInfo = {
-      address: token.id,
+      address: tokenAddress,
       decimals: Number(token.decimals),
       derivedETH: token.derivedETH,
       name: token.name,
       symbol: token.symbol,
       totalLiquidity: token.totalLiquidity,
+      uniqueId: tokenAddress,
       ...metadata,
     };
     allTokens[tokenAddress] = tokenInfo;
   });
+
+  // Explicitly add ETH token with WETH liquidity data
+  const wethToken = allTokens[WETH_ADDRESS];
```

Comment:
Since we are no longer relying strictly on the curated list to include ETH, I make sure to include ETH explicitly in the all tokens list.